### PR TITLE
Modifying test fields for the filebeat azure filesets

### PR DESCRIPTION
**Pull request by narph**
Should fix https://github.com/elastic/beats/issues/14185

The tests required that several properties be defined in the fields.yml conf file, this meant changes in the pipelines.
 
```
"claims" : {
                  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier" : "44a3b131"
...
}
```
`.` was replaced with `_` as it would create an object instead of a keyword.
- azure.auditlogs.properties.target_resources array flattened (same with the modified_properties)
- small renaming of properties to match naming standards

### CODE REVIEW

File: x-pack/filebeat/module/azure/activitylogs/test/activitylogs.log [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (exekias) [inline x-pack/filebeat/module/azure/activitylogs/test/activitylogs.log]: I'm wondering, shouldn't this be a single line raw JSON? as the pipeline is doing the decoding part

**Comment by narph** [comment]
jenkins test this

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -183,6 +183,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix calculation of `network.bytes` and `network.packets` for bi-directional netflow events. {pull}14111[14111]
 - Accept '-' as http.response.body.bytes in apache module. {pull}14137[14137]
 - Fix timezone parsing of MySQL module ingest pipelines. {pull}14130[14130]
+- Fix azure filesets test files.{issue}14185[14185] {pull}14235[14235]
```
Comment (kaiyan-sheng) [inline CHANGELOG.next.asciidoc]:
```suggestion
- Fix azure filesets test files. {issue}14185[14185] {pull}14235[14235]
```

**Comment by exekias** [comment]
@narph I see some of the pipelines changed. Could you please update the description to explain the changes?

**Comment by narph** [comment]
@exekias 

> I see some of the pipelines changed. Could you please update the description to explain the changes?
updated the description with the major changes

File: x-pack/filebeat/module/azure/auditlogs/_meta/fields.yml [hunk 3]
```diff
@@ -52,16 +56,17 @@
           type: keyword
           description: >
             ID
-        - name: activityDateTime
+        - name: activity_datetime
           type: keyword
```
Comment (kaiyan-sheng) [inline x-pack/filebeat/module/azure/auditlogs/_meta/fields.yml]: hmm do we have a date type for timestamps?

Comment (narph) [reply x-pack/filebeat/module/azure/auditlogs/_meta/fields.yml]: "date" , I have made the changes

**Pull request merged**